### ISSUE TITLE
Cluster.split (acg & dcg methods default parameters)

### ISSUE THREAD

**Issue by Setnea** [dizak/mothulity#43]
Due to the limitations of acg & dcg methods with default parameters (in complex analysis many bacterial genres are assigned as other bacteria i.e. proteobacteria), one must **consider** tuning the setup if the acg will remain the main clustering method.

The issue needs to be further discussed.

**Comment by dizak** [comment]
I agree. An actual analysis with different values of ```precision``` in the ```cluster.split``` command must be tested and benchmarked. One of the questions is whether the change of ```precision``` value does increase the RAM consumption significantly.

**Comment by dizak** [comment]
@Setnea Since the problem with *other bacteria* was recognized with #47 and fixed with #49 I think you can close the issue.
